Producers

```diff
@@ -383,19 +383,47 @@ type Global struct {
 	LastNameClose                  string  `json:"last_name_close"`
 }
 
+type ClaimTime struct {
+	eos.JSONTime
+}
+
+func (t ClaimTime) MarshalJSON() ([]byte, error) {
+	return []byte(fmt.Sprintf("%q", t.JSONTime.Format(eos.JSONTimeFormat))), nil
+}
+
+func (t *ClaimTime) UnmarshalJSON(data []byte) (err error) {
+	var unixNano eos.Int64
+	// With nodeos v1.4.4, the last_claim_time field is a Int64
+	// {"rows":[{"owner":"eoshuobipool","total_votes":"584497237807169280.00000000000000000","producer_key":"EOS5XKswW26cR5VQeDGwgNb5aixv1AMcKkdDNrC59KzNSBfnH6TR7","is_active":1,"url":"http://eoshuobipool.com","unpaid_blocks":228,"last_claim_time":"1545690481500000","location":0}],"total_producer_vote_weight":"28703068147968024576.00000000000000000","more":"eoslaomaocom"}
+	err = json.Unmarshal(data, &unixNano)
+	if err == nil {
+		t.JSONTime.Time = time.Unix(0, int64(unixNano))
+	} else {
```

Comment:
It would be preferable to check only if it was a `json.UnmarshalTypeError` only. So we do the fallback only if the type was wrong and not for any other unrelated errors.

```suggestion
	} else if _, isTypeError := err.(*json.UnmarshalTypeError); isTypeError {
```